packages/orchestrator-ui-components/src/pages/subscriptions/WfoSubscriptionsListPage.tsx

```diff
@@ -55,28 +48,25 @@ export const WfoSubscriptionsListPage = () => {
 
     const [activeTab, setActiveTab] = useQueryParam(
         'activeTab',
-        withDefault(StringParam, WfoSubscriptionsTabType.ACTIVE),
+        withDefault(StringParam, WfoSubscriptionListTab.ACTIVE),
     );
 
-    const sortOrder = getSortDirectionFromString(
-        dataDisplayParams.sortBy?.order,
-    );
-    const selectedSubscriptionsTab =
-        getSubscriptionsTabTypeFromString(activeTab);
-    if (!sortOrder || !selectedSubscriptionsTab) {
-        router.replace('/subscriptions');
-        return null;
-    }
+    const selectedTab = (): WfoSubscriptionListTab => {
```

Comment:
Suggesting to change to `getSelectedTab`

Reply:
Made it an IIFE so selectTab get's set right away